README.md

```diff
@@ -52,6 +54,17 @@ If you are new to Graviton and want to understand how to identify target workloa
 |DRAM	|8x DDR4	|8x DDR5	|
 |DDR Encryption	|yes	|yes	|
 
+# x86 monitoring differences
+## Utilities for obtaining processor stats
+|Processor	|x86	|Graviton2,3	|
+|---	|---	|---	|
+|CPU frequency listing	|*lscpu, /proc/cpuinfo, dmidecode*	|*dmidecode*	|
+|*turbostat* support	|Yes	|No	|
+|*i7z* Works	|Yes	|No	|
+|Performance monitoring tools	|_[VTune Profiler](https://www.intel.com/content/www/us/en/developer/tools/oneapi/vtune-profiler.html) and [PCM](https://github.com/opcm/pcm), [Linux perf](https://www.brendangregg.com/perf.html)_	|_[Linux perf](https://www.brendangregg.com/perf.html), [Arm Forge](https://developer.arm.com/Tools%20and%20Software/Arm%20Forge)_	|
```

Comment:
we have a section on perf monitoring already. if you want to put a comment there and maybe link to it